### ISSUE TITLE
APIs for authentication

### ISSUE THREAD

**Issue by ulhas** [proceduretech/drogon#4]
Finalise on the APIs for authentication

**Comment by san-coding** [comment]
SAWO API for user authentication can be a good choice

**Comment by ulhas** [comment]
@san-coding, yes. We could use a SAAS for this. But this is also meant to be a learning project. Would want to whoever is working on this project to also learn it.

Also, since this is open source, we never know how someone would want to implement authentication.
